app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def new
```

Comment:
- [ ] The Likes should be create in `create` action, so kinldy use `def create` method.

Reply:
Done